.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
do you need any of this? i.e why not just go with the default?

Reply:
It's mostly personal preference. I like semicolons and a tab with of 4. We can remove if you prefer.

Reply:
easier to use the defaults really so people can just be used to the same prettier on all projects.